### PR TITLE
Adding localforage and StorageService

### PR DESCRIPTION
**Pull request by Kyrrui**
This is a cleaned up version of #103 - this does not contain device management, only identity loading and storage in the browser. I plan on writing tests for this, but first I need to create a new [custom localforage driver](https://github.com/localForage/localForage/wiki/Custom-Drivers)

### CODE REVIEW

**Comment by Kyrrui** [comment]
I decided to take out device management from this PR, because there is another bounty being worked on that would obsolete this method of device management (3box storage)

File: universal-login-example/src/components/ContentContainer.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ class ContentContainer extends Component {
     this.state = {
       view: 'Login'
     };
+    this.loadIdentity();
```
Comment (marekkirejczyk) [inline universal-login-example/src/components/ContentContainer.js]: I propose to move it to `Services.start`

File: universal-login-example/src/services/StorageService.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    return await localforage.setItem(IDENTITY_KEY, identity, async (err) => {
+      if (err) {
+        return false;
+      }
+      return true;
+    });
+  }
+
+  async clearStorage() {
+    return await localforage.clear();
```
Comment (marekkirejczyk) [inline universal-login-example/src/services/StorageService.js]: Shorter version:  
```
async clearStorage() {
    return localforage.clear();
}
```
You can skip await if you return promise form async function.

**Comment by Kyrrui** [comment]
@marekkirejczyk Not sure what's up with the CI, but can't merge :/

**Pull request merged**